tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -82,12 +82,14 @@ public SpanContext(ulong? traceId, ulong spanId, SamplingPriority? samplingPrior
         /// <param name="samplingPriority">The propagated sampling priority.</param>
         /// <param name="serviceName">The service name to propagate to child spans.</param>
         /// <param name="origin">The propagated origin of the trace.</param>
-        internal SpanContext(TraceId traceId, ulong spanId, int? samplingPriority, string serviceName, string origin)
+        /// <param name="isRemote">Whether this <see cref="SpanContext"/> was from a distributed context.</param>
+        internal SpanContext(TraceId traceId, ulong spanId, int? samplingPriority, string serviceName, string origin, bool isRemote = false)
```

Comment:
I'm not a fan of optional parameters just because are you _sure_ you've updated everywhere you need to? 😄 Not opposed to it here, but I would still suggest (if you haven't already) on temporarily making `isRemote` a required parameter on all these APIs, and just double checking that we've definitely "fixed" all the ones we need to 😄

Reply:
I did do that initially, but I'll double check again just to be sure 👍

Reply:
Yeah I don't see any other instances where `isRemote` should be set to `true`